backend/server.py

```
@@ -7681,6 +7681,39 @@ async def clear_response_cache(user: dict = Depends(get_current_user)) -> dict[s
     return {"cleared": count}
 
 
+@app.get("/api/metrics/cost-attribution")
+async def cost_attribution_stats(user: dict = Depends(get_current_user)) -> dict[str, object]:
```

Ruff (0.15.21)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)